docs/training.mdx

```diff
@@ -0,0 +1,256 @@
+---
+title: "Training"
+description: "Methodology for calculating the normalized, amortized emissions from training AI models"
+---
+
+## Overview
+
+We need to model the environmental impact of the computation for the trained model. The training costs of genAI models are often disclosed in some form. These costs must be normalized and then amortized across the expected use-life of the model.
+
+The key components of the training cost of the model include:
+- The operational energy used to train the final model
+- The operational energy used to train intermediate and preliminary models
+- The embodied emissions from the cluster used to train all models, including hardware reserved but not actively deployed
+- (Allocated emissions from generation of the training and test data)
+- (Allocated operational and embodied emissions of development and testing infrastructure)
+
+[Carbon Emissions and Large Neural Network Training](https://arxiv.org/pdf/2104.10350) includes a table that shows the impact of changing the model, the datacenter, the gpu, and the energy grid in training a model:
+
+![Table of training costs](images/table1_carbon_emissions.png)
+
+## Disclosure of training costs
+
+To fully assess environmental impact, model developers should disclose the technical infrastructure used for training and how this infrastructure was engaged during the training process.
+
+Infrastructure data:
+- [Training cluster](/cluster) details
+- Training cluster location(s) - if hosted in a cloud, which region(s)
+
+Operational data:
+- Total reserved/owned wall clock time for intermediate and final model training
+- GPU hours for intermediate model training
+- GPU hours for final model training
+
+### Example disclosure
+
+As an example of a relatively complete disclosure, see [Sasha Luccioni, Sylvain Viguier, and Anne-Laure Ligozat. 2022. Estimating the carbon footprint of Bloom, a 176B parameter language model.](https://arxiv.org/pdf/2211.02001): "We estimate that BLOOM’s final training emitted approximately 24.7 tonnes of CO2eq if we consider only the dynamic power consumption, and 50.5 tonnes if we account for all processes ranging from equipment manufacturing to energy-based operational consumption."
+
+Infrastructure data
+
+| Component | Disclosed data |
+| --------- | -------------- |
+| GPU | Nvidia A100 80GB |
+| Server | HPE Apollo 6500 Gen10 Plus |
+| Number of GPUs | 384 |
+| Number of servers | 48 |
+| Training location | France |
+
+Training data
+
+| Component | Disclosed data |
+| --------- | -------------- |
+| Total reserved time | 118 days |
+| Reservation start time | January 2022 (?) |
+| GPU hours for final model | 1,082,990 |
```

Comment:
is GPU hours suppose to be the hours where the GPUs were used during the intermediate and final training? 

In this case with a cluster of 384 GPUs, this total GPU hours represents 117.5 (1082990/384/24) cluster days, which is the same as total reserved time.

I would have thought total reserved time in days to be more than the GPU hours for final model.

Reply:
Right - that's where we backfill the intermediate down below when we normalize. If you have ideas on how to clarify please lmk